Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -951,7 +951,17 @@ void vtkMRMLSliceNode::ReadXMLAttributes(const char** atts)
     // Layout color is set in Superclass
     layoutColorFound = true;
   }
+
+
   vtkMRMLReadXMLVectorMacro(fieldOfView, FieldOfView, double, 3);
+  // Adjust the FieldOfView to match the aspect ratio of the slice dimensions.
```

Comment:
Could we move these vtkMRMLSliceNode changes to a separate PR so they could be integrated now?

The changes made in ReadXMLAttributes are also applicable to CopyContent, and would fix slice rendering issues issues when replaying sequences of vtkMRMLSliceNode.

Reply:
sure, it makes sense. I am still working on this PR and It will take a while before being ready for review/merging.

Reply:
@Sunderlandkyl PR is here https://github.com/Slicer/Slicer/pull/8370, if you can test it would be great